plugins/com.jboss.devstudio.core.cheatsheets/src/com/jboss/devstudio/core/cheatsheet/Debug.java

```diff
@@ -0,0 +1,25 @@
+/*************************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ *    Red Hat, Inc. - Initial implementation.
+ ************************************************************************************/
+package com.jboss.devstudio.core.cheatsheet;
```

Comment:
@dgolovin It should be better to keep the package name same as plugin name, shouldn't it? Using plural form of cheatsheet (cheatsheets) everywhere (package name, manifest and others)...